extension-style-app/build.gradle.kts

```diff
@@ -31,10 +31,11 @@ android {
     }
   }
 
-  if (buildFromSource.toBoolean()) {
-    packagingOptions {
-      pickFirst("**/libc++_shared.so")
+  packagingOptions {
+    if (buildFromSource.toBoolean()) {
+      jniLibs.pickFirsts.add("**/libc++_shared.so")
     }
+    jniLibs.pickFirsts.add("**/libmapbox-common.so")
```

Comment:
why is it needed and even if it's needed - shouldn't it be inside `if` above?

Reply:
Without this config there will be two libmapbox-common.so and fails the build

Reply:
@Chaoba why was it working before? is it related to gradle update?

Reply:
No, it is related to this bump. Not exactly sure why.

Reply:
Could it be related to the gl-native gradle version bump? cc @ank27

Reply:
Actually this is a regression from gl-native, and it will break our customers as well, just downloaded the `android-core-10.4.0-beta.1.aar` and it contains `libmapbox-common.so`, which shouldn't be the case, since we have a separate common dependency that has the same `libmapbox-common.so`.

Reply:
Seen similar issues when switching between buildFromSource true / false, does it really reproduce with full clean build?

Reply:
> Could it be related to the gl-native gradle version bump? cc @ank27
It shouldn't be there. it could be related to the version bump but need to check further.